Example/Tests/UnsplashKit+CallsSpecs.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import Quick
+import Nimble
+import Mockingjay
+import RxSwift
+
+@testable import UnsplashKit
+
+class UnsplashKitCallsSpec: QuickSpec {
+    override func spec() {
+        
+        var subject: UnsplashKit!
+        var requestFactory: SourceRequestFactory!
+        var image: UIImage!
+        
+        func builder(request: NSURLRequest) -> Response {
```

Comment:
I wouldn't take data from this method that is out of the scope like the image. Instead, pass the image as a parameter:

```swift
func builder(request: NSURLRequest, image: UIImage) -> Response
```